azure-pipelines.yml

```diff
@@ -0,0 +1,486 @@
+trigger:
+- master
+
+variables:
+  - name: TeamName
+    value: dotnet-core-acquisition
+  # Skip Running CI tests
+  - name: SkipTests
```

Comment:
Do we run tests in our official build?  It seems odd to skip these otherwise.

Reply:
Yes,I noticed this too. Will set to 'false' by default.